### PR TITLE
Yhk dev

### PR DESCRIPTION
**Pull request by younghk**
review plz

### CODE REVIEW

File: Assets/CollisionAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{
+  // Start is called before the first frame update
+  void OnTriggerEnter2D(Collider2D other)
+  //rigidBody가 무언가와 충돌할때 호출되는 함수 입니다.
+  //Collider2D other로 부딪힌 객체를 받아옵니다.
+  {
+    if (other.gameObject.tag.Equals("feed"))
+    //부딪힌 객체의 태그를 비교해서 적인지 판단합니다.
+    {
+      // Destroy(other.gameObject);
```
Comment (giraffeJ) [inline Assets/CollisionAdapter.cs:15]: please delete not using lines

**Review by JeonYongHyun — APPROVED** [review]
LGTM

File: Assets/SceneTwoCharacterController.cs [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    if (Input.GetKey(KeyCode.D))
+    {
+      moveX += moveScale;
+    }
+
+    if (Input.touchCount > 0 || Input.GetMouseButton(0))
+    {
+      moveX += moveScale;
+    }
```
Comment (giraffeJ) [inline Assets/SceneTwoCharacterController.cs:54]: Can you merge this long scripts?

**Review by giraffeJ — APPROVED** [review]
LGTM

**Pull request merged**